### PR TITLE
feat(hermes): add uniqueness check for benchmark price 

### PR DESCRIPTION
**Pull request by ali-bahjati**

### CODE REVIEW

File: hermes/src/api/rest/get_vaa.rs [hunk 1]
```diff
@@ -97,13 +97,22 @@ pub async fn get_vaa(
...
-        .price_feed
-        .get_price_unchecked()
-        .publish_time;
+        .into_iter()
+        .next()
+        .ok_or(RestError::UpdateDataNotFound)?;
+
+    let publish_time = price_feed.price_feed.get_price_unchecked().publish_time;
+
+    // Currently Benchmarks API doesn't support returning the previous publish time. So we
```
Comment (m30m) [inline hermes/src/api/rest/get_vaa.rs:108]: if you expect the benchmark service to return 404 if not unique, why do we have this check here?

**Review by m30m — APPROVED** [review]
lgtm, added some minor comments

File: hermes/src/api/rest/get_price_feed.rs [hunk 2]
```diff
@@ -89,6 +89,15 @@ pub async fn get_price_feed(
         .next()
         .ok_or(RestError::UpdateDataNotFound)?;
 
+    // Currently Benchmarks API doesn't support returning the previous publish time. So we
```
[the remark this answers is not in the document]
Comment (ali-bahjati) [reply hermes/src/api/rest/get_price_feed.rs:92]: yeah i think they don't return it

File: hermes/src/api/rest/get_vaa.rs (hunk 1 above)
Comment (ali-bahjati) [reply hermes/src/api/rest/get_vaa.rs:108]: This applies to the cache of Hermes.

**Pull request merged**